### PR TITLE
Update cloudbuild.yaml

### PR DESCRIPTION
**Pull request by navierula**
Correct substitution variable name.

### CODE REVIEW

**Comment by dinagraves** [comment]
Hi Navie -- this is an artifact registry repo name, not the default substitution available from the GitHub trigger!  Maybe we can change the name to make that clearer?

**Comment by navierula** [comment]
Ah! :) Thanks for clarifying, Dina. Yes, let's change the name.

What do you think of "_REPO" or "_REPOSITORY_NAME"?

**Comment by glasnt** [comment]
I'd suggest `_REGISTRY_NAME`. 
In my head, "repo" means "repository", not "registry", so being explicit might be useful in this example/

**Comment by dinagraves** [comment]
> I'd suggest `_REGISTRY_NAME`. In my head, "repo" means "repository", not "registry", so being explicit might be useful in this example.

Should it be _ARTIFACT_REGISTRY_REPO ?  Since they are [called repos](https://cloud.google.com/artifact-registry/docs/manage-repos)?

**Comment by glasnt** [comment]
> > I'd suggest `_REGISTRY_NAME`. In my head, "repo" means "repository", not "registry", so being explicit might be useful in this example.
> 
> Should it be _ARTIFACT_REGISTRY_REPO ? Since they are [called repos](https://cloud.google.com/artifact-registry/docs/manage-repos)?

TIL 😅 

Yes, `_ARTIFACT_REGISTRY_REPO`

**Comment by navierula** [comment]
I updated the file with the new variable name, but the test seems to be failing. Any suggestions?

**Comment by glasnt** [comment]
I've updated the substitution variable `_REPO_NAME` to `_ARTIFACT_REGISTRY_REPO` on the trigger configuration itself, which should resolve the error.

File: python-example-flask/cloudbuild.yaml [hunk 1]
```diff
@@ -59,5 +59,5 @@ artifacts:
 # [END cloudbuild_python_logs_yaml]
 # Store images in Google Artifact Registry
```
Comment (glasnt) [inline python-example-flask/cloudbuild.yaml:60]:
```suggestion
# Store images in Google Artifact Registry 
```

**Review by dinagraves — APPROVED** [review]

**Comment by navierula** [comment]
Thanks for the review and updates!

**Pull request merged**